extensions/webapps/consent-app/src/ConsentPage.css

```
@@ -0,0 +1,239 @@
+  --text-muted: #73738c;
+  --danger: #d32f2f;
+  --danger-light: #fff5f5;
+  --radius: 8px;
+  --radius-lg: 12px;
+  --shadow: 0px 2px 1px -1px rgba(21, 21, 21, 0.08), 0px 1px 1px 0px rgba(21, 21, 21, 0.06), 0px 1px 3px 0px rgba(21, 21, 21, 0.04);
+}
+
+.consent-page {
+  min-height: 100vh;
+  background: var(--background);
+  font-family: 'Gilmer', 'Segoe UI', Helvetica, Arial, sans-serif;
```

Stylelint (17.9.0)

[error] Expected no quotes around "Gilmer" (font-family-name-quotes)

(font-family-name-quotes)